ee/backup/run.go

```diff
@@ -131,7 +131,7 @@ $ dgraph restore -p . -l /var/backups/dgraph -z localhost:5080
 
 	flag.StringVarP(&opt.badger, "badger", "b", worker.BadgerDefaults,
 		z.NewSuperFlagHelp(worker.BadgerDefaults).
-			Head("Badger options").
+			Head("Badger options (Other badger options are also supported, not shown for brevity.)").
```

Comment:
refer to badger documentation for all possible options. don't add link but add in readme.